modules/nodes-pods-autoscaling-custom-metrics-access.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * nodes/pods/nodes-pods-autoscaling-custom.adoc
+// Modeled after migration-accessing-performance-metrics.adoc
+
+:_content-type: PROCEDURE
+[id="nodes-pods-autoscaling-custom-metrics-access_{context}"]
+= Accessing performance metrics
+
+The Custom Metrics Autoscaler Operator records metrics and pulls them into on-cluster monitoring storage. You can query the metrics by using Prometheus Query Language (PromQL) to diagnose migration performance issues. All metrics are reset when the Migration Controller pod restarts.
+
+You can access the performance metrics and run queries by using the {product-title} web console.
```

Comment:
Not sure all metrics are "performance metrics".

Reply:
I agree.